README.md

```diff
@@ -1,3 +1,26 @@
 # [GeoCAT Applications](https://ncar.github.io/geocat-applications/)
 
 This is the source repository for GeoCAT-applications.
+
+## Contributors
+Thank you to all of our contributors ([emoji key](https://allcontributors.org/docs/en/emoji-key)):
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+<table>
+  <tbody>
+    <tr>
+      <td align="center" valign="top" width="14.28%"><a href="https://github.com/anissa111"><img src="https://avatars.githubusercontent.com/u/38434768?v=4?s=75" width="75px;" alt="Anissa Zacharias"/><br /><sub><b>Anissa Zacharias</b></sub></a><br /><a href="#infra-anissa111" title="Infrastructure (Hosting, Build-Tools, etc)">🚇</a> <a href="https://github.com/NCAR/geocat-applications/commits?author=anissa111" title="Code">💻</a> <a href="#ideas-anissa111" title="Ideas, Planning, & Feedback">🤔</a></td>
```

Comment:
I agree the code vs docs thing is murky, but this seems like a great start!

Thanks for putting this all together!